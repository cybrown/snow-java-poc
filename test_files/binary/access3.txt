a.property[0] + ("12" + 3)