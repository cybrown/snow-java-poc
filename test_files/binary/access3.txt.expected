<Binary operator="+">
    <Binary operator="[">
        <Binary operator=".">
            <Identifier>a</Identifier>
            <Identifier>property</Identifier>
        </Binary>
        <LiteralInteger>0</LiteralInteger>
    </Binary>
    <Binary operator="+">
        <LiteralString>"12"</LiteralString>
        <LiteralInteger>3</LiteralInteger>
    </Binary>
</Binary>
